packages/core/src/components/spinner/_spinner.scss

```diff
@@ -211,7 +216,11 @@ $resources-path: "./resources" !default;
   }
 }
 
-@keyframes pt-spinner-animation {
-  from { transform: rotate(0deg); }
-  to   { transform: rotate(360deg); }
+.pt-input-group .pt-spinner {
```

Comment:
Usually we'd put this in `_input-group.scss`, feels weird to see this rule here. Any particular reason why? Having all the modifiers and compositions for a component in the same component file (excluding commons) seems easier to maintain